auth-web/src/components/auth/create-account/AccountCreatePremium.vue

```diff
@@ -242,6 +243,7 @@ export default class AccountCreatePremium extends Mixins(Steppable) {
       bcolAccountName: details.bcolAccountDetails.orgName
     }
     this.setCurrentOrganization(org)
+    this.setCurrentOrganizationPaymentType(PaymentTypes.BCOL)
```

Comment:
Makes sense to set payment to BCOL, considering the user just typed in their USER / pass to fetch their previous account details.